### PR TITLE
G640s Fix and Express Key support

### PR DESCRIPTION
**Pull request by skyborgff**
`d.GetMaxOutputReportLength() != 0`
Adding this doesn't break any of my tablets. idk if it may break others, but this allows to select the correct HID.

ActiveReportID while hovering is 80, so i set it to 79 so it can pass this check https://github.com/InfinityGhost/OpenTabletDriver/blob/6dbf1e740a4c6f66f3e487714e17455ae392016b/TabletDriverLib/Output/AbsoluteMode.cs#L89

### CODE REVIEW

File: TabletDriverLib/Driver.cs [hunk 1]
```diff
@@ -34,7 +34,7 @@ public bool Open(TabletProperties tablet)
             try
             {
                 var matching = Devices.Where(d => d.ProductID == tablet.ProductID && d.VendorID == tablet.VendorID);
-                var tabletDevice = matching.FirstOrDefault(d => d.GetMaxInputReportLength() == tablet.InputReportLength);
+                var tabletDevice = matching.FirstOrDefault(d => d.GetMaxInputReportLength() == tablet.InputReportLength & d.GetMaxOutputReportLength() != 0);
```
Comment (InfinityGhost) [inline TabletDriverLib/Driver.cs:37]: This shouldn't break any detection, this is fine

File: TabletDriverLib/Vendors/XP_Pen/XP_PenAuxReport.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        {
+            Raw = report;
+            var ReportID = (uint)report[1] >> 1;
+            var ButtonInt = (uint)report[2];
+            AuxButtons = new bool[6];
+            if (ReportID == 120) 
+            {
+                AuxButtons = new bool[] 
+                {
+                ButtonInt == 1,
```
Comment (InfinityGhost) [inline TabletDriverLib/Vendors/XP_Pen/XP_PenAuxReport.cs:17]: Fix indent L17-22

File: TabletDriverLib/Configurations/XP-Pen/G640 V2.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+  "Name": "XP-Pen G640 v2",
```
Comment (InfinityGhost) [inline TabletDriverLib/Configurations/XP-Pen/G640 V2.json:2]: Remove this config, we'll rely on #134 for this

Comment (skyborgff) [reply TabletDriverLib/Configurations/XP-Pen/G640 V2.json:2]: shouldnt this disapear, as it was merged, and this is now part of the master?

Comment (InfinityGhost) [reply TabletDriverLib/Configurations/XP-Pen/G640 V2.json:2]: Update branch from master

**Pull request merged**